src/util/serializing/ObjectOutputStream.cpp

```diff
@@ -60,6 +60,7 @@ void ObjectOutputStream::writeData(const void* data, int len, int width) {
     }
 }
 
+[[ maybe_unused ]]
```

Comment:
This function can be remove (in a separate commit), I think. I can't imagine it being back in our good graces in the future.